website/content/en/docs/User Guide/Cost Explained/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Breaking Down Cloud Operations Sandbox Cost"
+linkTitle: "Sandbox Cost Explained"
+weight: 2
+---
+{{% pageinfo %}}
+* [Cloud Operations Sandbox Cost Estimations](#Cloud-Operations-Sandbox-Cost-Estimations)
+  * [GCP Calculator](#GCP-Calculator)
+  * [Sandbox Cost Estimations](#Sandbox-Cost-Estimations)
+  * [Sandbox Compute Cost Estimations](#Sandbox-Compute-Cost-Estimations)
+  
+ 
+## Cloud Operations Sandbox Cost Estimations
+Cloud Operations Sandbox Cost is combined from compute cost, storage and usage of network and additional services like Cloud Operations Suite. While some of the cost, like GKE infrastructure is constant, for some services cost is a factor of usage. Note, that some of Sandbox costs fall under the [free tier](https://cloud.google.com/free/docs/gcp-free-tier#free-tier), but not not all.
+To learn more about billing and pricing please refer to: [Google Cloud Pricing Overview](https://cloud.google.com/pricing/), [Google’s Pricing philosophy](https://cloud.google.com/pricing/philosophy/), [GCP's Billing](https://cloud.google.com/billing/docs).
+
+To view billing information for your project in the console go to [billing.](https://console.cloud.google.com/billing)
+![image](/docs/images/user-guide/57-billing-menu.png)
+
+![image](/docs/images/user-guide/58-billing-console.png)
+ 
+### GCP Calculator
+GCP has transparent pricing which is applied per resource. You can plan and estimate pricing using the [pricing calculator](https://cloud.google.com/products/calculator). Some of the really cool features of Google cloud platform are sustained use discounts that are applied as credits to your account rather than just lower costs. That allows us later to see exact costs attributed to usage, so we can more accurately reduce usage later.
+
+### Sandbox Cost Estimations
+To break down the cost we will use a GCP calculator. As network, storage and [observability cost](https://cloud.google.com/stackdriver/pricing) are very minimal due to the low volume, in the next section we will focus on the cost of the infrastructure. 
```

Comment:
1. Is it true that networking is minimal cost? Last I checked, public GKE loadbalancers cost a flat $30 or something
2. I'm not sure if this section is needed, it seems like this can be merged with the one below

Reply:
Changed to 'network traffic' is it clearer?

Reply:
Yeah I think so. But I do think load balancer cost should be shown somewhere if possible